### ISSUE TITLE
About testing on videos with pre-trained model

### ISSUE THREAD

**Issue by KobayashiLiu** [tangyuhao/GOTURN-Tensorflow#24]
Hello.When I using your pre-trained model to test on video, there is a problem that, the bounding box shifts away seriously from the object I want to track.And the target isn't at the center of the target_pad.Then I find your pre-trained model can make the loss to 15~50 using the data you provide.
I wanna know if the pre-trained model is not good enough, or some other reasons.
Thanks

**Comment by xujiao3628** [comment]
> 你好。当我使用你预先训练过的模型来测试视频时，有一个问题是，边界框会严重偏离我想要跟踪的对象。目标不在target_pad的中心。然后我找到您预先训练的模型可以使用您提供的数据将损失降至15~50。
> 我想知道预先训练好的模型是不够好还是其他一些原因。
> 谢谢

你好，能给我发一下预训练模型吗，谢谢你啦！1763165393@qq.com

**Comment by KobayashiLiu** [comment]
@xujiao3628 https://pan.baidu.com/s/1QYMdyGuuDZ992DIJZskdfA 
code:85gu

**Comment by xujiao3628** [comment]
> @ xujiao3628 https://pan.baidu.com/s/1QYMdyGuuDZ992DIJZskdfA
> 代码：85gu

非常感谢
